NCU-Project/API/DogsNetworkingAPI.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  DogsNetworkingAPI.swift
+//  NCU-Project
+//
+//  Created by Simon Rowlands on 19/10/2018.
+//  Copyright © 2018 simonrowlands. All rights reserved.
+//
+
+import Foundation
+
+import RxSwift
+
+final class DogsNetworkingAPI: BaseNetworkingAPI {
+    
+    static let baseURL = BaseURL("https://dog.ceo/api")
+    
+    static let allDogsList = "/breeds/list/all"
```

Comment:
is it worth extracting the path strings you have dotted around into a enum/struct so they are all in one place?